docs/source/guide/calibrators.md

```diff
@@ -1,12 +1,13 @@
 ---
 jupytext:
+  formats: ipynb,md:myst
   text_representation:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.14.1
+    jupytext_version: 1.16.1
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: mitiq
```

Comment:
Let's keep this as is just since all our other docs have `Python 3` or `Python 3 (ipykernel)`.

Reply:
This was set back to `Python 3 (ipykernel)`  in https://github.com/unitaryfund/mitiq/pull/2568/commits/321a4f19961034049cc2c5a219aeda683215a092